Makefile

```diff
@@ -41,6 +41,23 @@ $(TOOLS)/kubectl: $(TOOLS)
 .PHONY: tools
 tools: $(JQ) $(YQ) $(KUBECTL)
 
+OTEL_VERSION?=$(COLLECTOR_CONTRIB_VERSION)
+.PHONY: update-otel-version
+update-otel-version:
+	OLD_VERSION=$(shell grep -Eo 'COLLECTOR_CONTRIB_VERSION=[0-9.]+' VERSION | cut -d'=' -f2); \
+	sed -i "s|otel/opentelemetry-collector-contrib:$${OLD_VERSION}|otel/opentelemetry-collector-contrib:$(OTEL_VERSION)|g" config/*; \
```

Comment:
How critical is it that we only update `OLD_VERSION`? What if we relax this to something like `sed -i "s|otel/opentelemetry-collector-contrib:[0-9.]+|otel/opentelemetry-collector-contrib:$(OTEL_VERSION)|g" config/*;`? It'd also remove the need to query `OLD_VERSION`.

Reply:
You're right, updated it to make things simpler.